### ISSUE TITLE
[Pods 2.8 Bug] Images not saving correctly

### ISSUE THREAD

**Issue by AndyRaphael** [pods-framework/pods#6089]
## Pods 2.8 version

2.8 b5

## Description

Prior to upgrading from 2.7, I had images in my WYSIWYG fields. All those images are gone. Also, I have a multi image field. I can add pictures and save and the post will show correctly. However, when I go back in to edit the Pod again, the images are gone and when I update the post, they also are gone from the post.

## Testing instructions

1. Add an image to a multi image field
2. Publish post
3. Leave pod and then come back to edit the post
4. Images are gone

## Expected behavior

Images should not be removed upon reloading

## Possible workaround

Select images again and publish and then don't reload. Select images each time before publishing post

## Other super helpful information

<details>
<summary>WordPress Environment</summary>
  
```
Copy and paste your Tools > Site Health > Info. Use the 'Copy Site Info to Clipboard' and paste the results over this line.
```

</details>

<details>
<summary>Pods Package Export</summary>

```json
Copy and paste your Pods Admin > Migrate: Packages > Export and paste the code over this line.
```

</details>

**Comment by pdclark** [comment]
In testing, the WYSIWYG field worked for me, and the back-end editor showed images as selected, but front-end access to the field isn't showing anything when using `->field()`.

![Screen Shot 2021-08-25 at 9 53 19 AM](https://user-images.githubusercontent.com/126889/130832916-23041df4-4bc1-40fa-8bc3-ea158667af81.png)

**Comment by AndyRaphael** [comment]
Just to rule out anything in my theme or WP instance, I created a brand new WP instance. I only added the latest Pods 2.8 beta plugin (there are no there plugins). Created a basic Pod and it has the same issue as above. Pictures and sometime text fields are blank after updating.

**Comment by sc0ttkclark** [comment]
I'm seeing issues when WYSIWYG is present, I need to dig further into what's happening there.

**Comment by sc0ttkclark** [comment]
Fixed WYSIWYG issue with removing all HTML tags. That's now resolved via 5e60a54eb58edfb6dfbcbbfdcc79ed8b66ca1072
